src/services/codefixes/importFixes.ts

```diff
@@ -356,6 +357,24 @@ function createImportAdderWorker(sourceFile: SourceFile | FutureSourceFile, prog
         }
     }
 
+    function addImportForExternalModuleSymbol(symbol: Symbol, originalSymbol: Symbol, isValidTypeOnlyUseSite: boolean) {

```

Comment:
`symbol`/`originalSymbol` are confusing names here. It looks like `originalSymbol` is the actual module symbol, and `symbol` is just an existing alias for the module that’s only being used to see what name to use. Instead, try just passing in the module symbol and the name you want to use?